src/components/content/survey-table/survey-table.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import './survey-table.scss';
+
+
+class SurveyTable extends React.Component {
+    render() {
+        let classAnimationName = '';
+        if (this.props.opened === 'open') classAnimationName = 'survey-table-down ';
+        if (this.props.opened === 'close') classAnimationName = 'survey-table-up ';
+        return (
+            <div className={classAnimationName + 'survey-table'}>
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for string composition
```js
`${classAnimatonName} survey-table`
```